### DISCUSSION TITLE
unable to show python doc using "Shift-k"

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-nvim#71] [Q&A]
I'm unable to use `shift-k` to `show python doc` of a module, e.g. numpy or matplotlib, in qmd file. Also, keybinding `gd` for `goto definition` is also not working in python code in qmd file. 

I got the following error:
![image](https://github.com/quarto-dev/quarto-nvim/assets/23435810/ae79e43f-14ba-4176-9306-a0234a00b4d2)

I'm expecting the following when using `shift-k` in a normal python file:
![image](https://github.com/quarto-dev/quarto-nvim/assets/23435810/497bf6b5-a086-4756-abae-38a7040f96b9)

I understand that the following is mentioned here in this repo. However, I'm still not familiar with how to config the plugins related to lsp and code completion, etc. 
```
Because Quarto provides a lot of functionality through integration with existing plugins, some of those have to be told about the existence of quarto-nvim (like e.g. registering it as a source for the autocompletion plugin nvim-cmp).

As such, we recommend you to experiment with the [quarto-nvim kickstarter configuration](https://github.com/jmbuhr/quarto-nvim-kickstarter) and then pick the relevant parts from the [lua/plugins/quarto.lua](https://github.com/jmbuhr/quarto-nvim-kickstarter/blob/main/lua/plugins/quarto.lua) file to integrate it into your own existing configuration. 
```

So, can anyone provide some suggestions on how to edit the nvim plugins, such as `nvim-lspconfig`, `nvim-cmp`, etc. that are related to `quarto-nvim` so that we can get the same functions, such as auto-completion, etc when we edit python code in qmd files like we have when editing native python code in neovim? 

My `nvim` config with `quarto-nvim` installed is shown [here](https://github.com/hopezh/oatLazyVim).

**Comment by jmbuhr** [comment]
I can only recommend to go through https://github.com/jmbuhr/quarto-nvim-kickstarter/blob/main/lua/plugins/quarto.lua line by line to see how plugins that somehow interact with quarto can or should be configured.

**Reply by jmbuhr** [reply]
https://github.com/quarto-dev/quarto-nvim#setup
